### PR TITLE
chunkserver: do not init and expose walpool when not use curve protoc…

### PR DESCRIPTION
**Pull request by xu-chaojie**
…ol in raftLogUri

### What problem does this PR solve?

Issue Number: close #xxx 

Problem Summary:

### What is changed and how it works?

What's Changed:

How it Works:

Side effects(Breaking backward compatibility? Performance regression?): 

### Check List

- [x] Relevant documentation/comments is changed or added
- [x] I acknowledge that all my contributions will be made under the project's license

### CODE REVIEW

File: src/chunkserver/chunkserver.cpp [hunk 1]
```diff
@@ -138,20 +140,25 @@ int ChunkServer::Run(int argc, char** argv) {
...
-        kWalFilePool = std::make_shared<FilePool>(fs);
-        LOG_IF(FATAL, false == kWalFilePool->Initialize(walFilePoolOptions))
-            << "Failed to init wal file pool";
-    } else {
-        kWalFilePool = chunkfilePool;
-        LOG(INFO) << "initialize to use chunkfilePool as walpool success.";
+    std::string raftLogUri;
+    LOG_IF(FATAL, !conf.GetStringValue("copyset.raft_log_uri", &raftLogUri));
+    std::string raftLogProtocol = UriParser::GetProtocolFromUri(raftLogUri);
+    bool useChunkFilePoolAsWalPool = true;
```
Comment (cw123) [inline src/chunkserver/chunkserver.cpp:146]: 这字段什么意思，为true的时候表示什么

Comment (xu-chaojie) [reply src/chunkserver/chunkserver.cpp:146]: 为true表示walpool共用chunkfilepool

File: src/chunkserver/chunkserver.cpp [hunk 2]
```diff
@@ -138,20 +140,25 @@ int ChunkServer::Run(int argc, char** argv) {
...
+    if (raftLogProtocol == kProtocalCurve) {
+        LOG_IF(FATAL, !conf.GetBoolValue(
+            "walfilepool.use_chunk_file_pool",
+            &useChunkFilePoolAsWalPool));
+
+        if (!useChunkFilePoolAsWalPool) {
+            FilePoolOptions walFilePoolOptions;
+            InitWalFilePoolOptions(&conf, &walFilePoolOptions);
+            kWalFilePool = std::make_shared<FilePool>(fs);
+            LOG_IF(FATAL, false == kWalFilePool->Initialize(walFilePoolOptions))
```
Comment (ilixiaocui) [inline src/chunkserver/chunkserver.cpp:156]: outputs info log if succeeds

Comment (xu-chaojie) [reply src/chunkserver/chunkserver.cpp:156]: done

**Review by cw123 — APPROVED** [review]

**Review by ilixiaocui — APPROVED** [review]

**Pull request merged**